script.lua

```diff
@@ -2228,15 +2228,32 @@ function PickPower(cardo,playero,alt_click)
     if cardo.hasTag("Major") then
         Player[playero].broadcast("Don't forget to Forget a Power Card!", Color.SoftYellow)
     end
-    -- Give card to player regardless of whose hand they are in front of
+    -- Figure out which player the card is in front of
+    local handPos = nil
+    for color,_ in pairs(playerTables) do
```

Comment:
for iterating through a list of colors `selectedColors` has how i've typically been handling that

Reply:
I went with `playerTables` instead of `selectedColors` because all the power gain functionality still works even for a player who's picked a seat but not a spirit, in which case they appear in `playerTables` but not `selectedColors`. Sure, people probably *shouldn't* be gaining power cards before picking a spirit, but they *can* do, and it seems better to make it robust in case they do.

Reply:
interesting, fair point